python/social_bot/tasks.py

```diff
@@ -104,14 +104,16 @@ class GoalTask(Task):
 
     def __init__(self,
                  env,
-                 max_steps=500,
+                 max_steps=200,
```

Comment:
Default max_steps of Goaltask is changed to 200 which is more reasonable for the size of playground. 500 seems too large.